packages/metro-config/src/ExpoMetroConfig.ts

```diff
@@ -217,13 +217,14 @@ export interface LoadOptions {
   reporter?: Reporter;
   resetCache?: boolean;
   target?: ProjectTarget;
+  withoutDefaults?: boolean;
```

Comment:
Maybe supply `defaultConfig: object` which you could pass an empty object too, that would be used in favor of the `getDefaultConfig`.